### DISCUSSION TITLE
In lotus-bench sealing, why window-post takes longer than winning-post?

### DISCUSSION THREAD

**Discussion by Zengdewei1** [filecoin-project/lotus#6843] [General Q&A] [answered]
When I executed command `lotus-bench sealing --sector-size 32GiB`, I found that window-post takes 4 minutes while winning-post takes 2 seconds. Can anyone explain why?
my log is below.
![avatar](https://user-images.githubusercontent.com/40319959/126466644-cfef4839-92b7-4e4a-b2d2-fe74ee7c8f89.png)

**Comment by de-authority** [comment, marked as answer]
The calculation process is completely different. It is normal to spend different time doing different things. If you are interested, you can look at the source code, but I don't think it's necessary

**Reply by Zengdewei1** [reply]
I agree that they are different but I do feel confused. In winning, 66 leaves are selected. However, in window, 10 leaves are selected. But I do agree that circuit of window is more complicated than winning. So my problem is, why so different though they are based on post?

**Reply by de-authority** [reply]
Lol. i can't explain why they are different  in details. maybe you can found out in code. I will post you if I find docs that can explain this.

**Reply by Zengdewei1** [reply]
thanks. Hope to communicate more.

**Reply by de-authority** [reply]
You are welcome at any time.

**Reply by Kubuxu** [reply]
In short WindowPoSt is designed to prove up to 2300 sectors, for WinningPoSt it is way less.
